code_doc_gen/credential_manager.py

```
@@ -0,0 +1,63 @@
+        """
+        Retrieve a stored API key.
+        
+        Args:
+            provider: The AI provider name
+            
+        Returns:
+            The API key or None if not found
+        """
+        try:
+            return keyring.get_password(SERVICE_NAME, f"{provider}_api_key")
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)